doc/obs/docker.md

```
@@ -0,0 +1,7 @@
+# just docker things
+
+### set shm size for mpi
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)